src/env/mpicc.bash.in

```diff
@@ -184,6 +185,10 @@ for arg in "$@" ; do
     nativelinking=yes
     addarg=no
     ;;
+    -lcuda|-lcudart)
+    delay_libs="$delay_libs $arg"
+    addarg=no
+    ;;
```

Comment:
This commit probably snuck in from another branch.

Reply:
Indeed. Good catch!